packages/web/src/scss/components/Stack/README.md

```diff
@@ -39,14 +39,17 @@ Usage with a list:
 
 ## Variants
 
-⚠ `Stack--hasSpacing` uses the CSS `padding` property to be able to create dividers using the `border` property.
+⚠ `Stack--hasSpacing` with dividers uses the CSS `padding` property to be able to create dividers using the `border` property.
 It is recommended to wrap the direct descendants of the `Stack` component to the proper HTML element.
 Otherwise, the applied spacing via vertical padding could break the visual view of the children's items.
```

Comment:
We'd like to set the two scenarios more apart:

1. With spacing, don't wrap child items (don't mention it at all).
2. With dividers, wrapping is mandatory.

We could make the second scenario even clearer by providing a mandatory`StackItem` subcomponent:

```html
<Stack hasDividers>
  <StackItem>…</StackItem>
  <StackItem>…</StackItem>
  <StackItem>…</StackItem>
</Stack>
```

@janicekt Please do feel free to comment on this! 🙂

Reply:
**UPDATE:** This could be the way, but we decided to discuss it first this Thursday. @janicekt can join us of course 🙂.

Reply:
I am closing this conversation because if we add this, we will do it in another PR.